settings.gradle.kts

```diff
@@ -100,12 +104,18 @@ fun property(name: String) =
 
 val isCiServer = System.getenv().containsKey("CI")
 
-if (isCiServer) {
-    gradleEnterprise {
-        buildScan {
-            termsOfServiceUrl = "https://gradle.com/terms-of-service"
-            termsOfServiceAgree = "yes"
-            tag("CI")
+gradleEnterprise {
```

Comment:
Really, you have three scenarios here:

1. CI builds - With this PR, nothing changes regarding publishing of CI builds, except that they go to https://ge.apache.org. Your builds will pick up credentials automatically in Jenkins and GitHub.
2. Authenticated Local builds - Apache committers _can_ publish local build scans to https://ge.apache.org by using LDAP credentials to login and create an access token. If that is done, then local scans are published.
3. Unauthenticated Local builds - For these, the plugin will be applied, but since the user is not authenticated to https://ge.apache.org, nothing actually happens.

So, to summarize, you won't see any real differences to _when_ a build scan gets published, unless an Apache committer explicitly opts in to the local scans.